ppanggolin/utils.py

```diff
@@ -580,8 +583,8 @@ def combine_args(args: argparse.Namespace, another_args: argparse.Namespace):
     return args
 
 
-def get_args_that_differe_from_default(default_args: argparse.Namespace, final_args: argparse.Namespace,
-                                       param_to_ignore: Union[List[str], Set[str]] = None) -> dict:
+def get_args_that_different_from_default(default_args: argparse.Namespace, final_args: argparse.Namespace,
```

Comment:
I think I still prefer the original name of the function. 😔 
Otherwise we could go with `get_args_differing_from_default` ? 😀

Reply:
`get_args_differing_from_default` ?